server/src/main/java/org/apache/uniffle/server/buffer/ShuffleBufferManager.java

```diff
@@ -189,6 +189,20 @@ private void updateShuffleSize(String appId, int shuffleId, long size) {
     shuffleIdToSize.get(shuffleId).addAndGet(size);
   }
 
+  public boolean checkIfRegistered(String appId, int shuffleId, List<Integer> partitionIds) {
```

Comment:
Could we use ` ShuffleTaskManager#shuffleTaskInfos`  to judge whether is registered?

Reply:
+1 I have similar thought. The appId + shuffleId check existence are enough. Right?